spec/lib/mercury/monadic_spec.rb

```diff
@@ -362,32 +362,32 @@ def push_and_ack(array)
   describe '#source_exists?' do
     itt 'returns false when the source does not exist' do
       test_with_mercury do |m|
-        m.source_exists?('asdf').
-          and_lift { |result| expect(result).to be false }
+        m.source_exists?('asdf')
+         .and_lift { |result| expect(result).to be false }
```

Comment:
This transform is fine. It just bothers me every time I write chained method calls that ruby's parser at the end of every complete expression has to look ahead to the next line to see if it starts with a `.` :)